Packs/CommonPlaybooks/Playbooks/playbook-Search_For_Hash_In_Sandbox_-_Generic.yml

```diff
@@ -1242,24 +1242,70 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "45":
+    id: "45"
+    taskid: 834196a2-ca30-4f26-825c-d41abac9ea26
+    type: condition
+    task:
+      id: 834196a2-ca30-4f26-825c-d41abac9ea26
+      version: -1
+      name: Any results from the Falcon Intelligence Sandbox search?
+      description: Any results from the Falcon Intelligence Sandbox search?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "40"
+      "yes":
+      - "13"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: csfalconx.resource.FindReport
```

Comment:
Why does the conditional task check if 'csfalconx.resource.FindReport.foundIds' is not empty, when the next task uses the path 'csfalconx.resource.resources'?